test/topology_custom/test_mv_fail_building.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import asyncio
+import pytest
+import time
+from test.topology.conftest import skip_mode
+from test.pylib.manager_client import ManagerClient
+
+from cassandra.cluster import ConsistencyLevel  # type: ignore
+from cassandra.query import SimpleStatement  # type: ignore
+
+
+async def wait_for_view(cql, node_count):
```

Comment:
How about taking the name of the view as a parameter and checking only that view? We aren't interested in checking the status of other views. I know that there aren't supposed to be other views during the test, but this makes the function potentially more reusable in the future.

Reply:
I added the name parameter in the rebase